models/intel/person-detection-0200/description/person-detection-0200.md

```diff
@@ -8,24 +8,20 @@ prior boxes for 256x256 resolution.
 
 ## Example
 
-![](./person-detection-0100.png)
+![](./person-detection-0200.png)
 
 ## Specification
 
 | Metric                          | Value                                     |
 |---------------------------------|-------------------------------------------|
-| AP                              | 82.52% (internal test set)                |
-| Pose coverage                   | Standing upright, parallel to image plane |
-| Support of occluded persons     | YES                                       |
-| Occlusion coverage              | <50%                                      |
-| Min person height               | 100 pixels (on 1080p)                     |
+| AP @ [ IoU=0.50:0.95 ]          | 0.244 (internal test set)                 |
 | GFlops                          | 0.786                                     |
 | MParams                         | 1.817                                     |
```

Comment:
Are you sure, that these values has no changes?

Reply:
yes, there are no architectural changes. I also checked it.